src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +34,36 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo &#35;{selectedTodoId}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Isn't good to disable eslint rules, they can help to delivery a better application. In this case, your app can be more acessible.

Try to remove this line, if you are getting an error, please ask for help on the chat. =)

Reply:
This one came within the forked code... remove it anyway?